x-pack/plugins/security_solution/public/common/components/grouping/types.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { GenericBuckets } from '../../../../common/search_strategy/common';
+
+export const DEFAULT_GROUPING_QUERY_ID = 'defaultGroupingQuery';
+
+export type RawBucket = GenericBuckets & {
```

Comment:
Ahh, nice! I wonder if we should start to prepend these `RawBucket` types like `AlertGroupingRawBucket` or similar as we may end up having multiple `RawBucket` types that are for different situations, but not that easy to reason about.